src/app/components/share/facebook-script.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable */
+(function(d, s, id) {
+  var js, fjs = d.getElementsByTagName(s)[0];
+  if (d.getElementById(id) || !fjs) return;
+  js = d.createElement(s); js.id = id;
+  js.src = '//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.8';
+  fjs.parentNode.insertBefore(js, fjs);
+}(document, 'script', 'facebook-jssdk'));
+/* eslint-enable */
```

Comment:
I don't see this being used in the PR?

Reply:
It's [imported](https://github.com/openstax/os-webview/blob/port-components-to-ts-5/src/app/components/share/share.tsx#L7) by the "share" component.